src/utils/masonryDetails.js

```diff
@@ -8,41 +8,46 @@ const masonryDetails = {
       rowStart: 1,
       numberOfRows: 1,
     },
-    'Aria Attributes': {
-      rowStart: 'auto',
-      numberOfRows: 1,
+    'ARIA Roles and Attributes': {
+      rowStart: 2,
+      numberOfRows: 3,
+      columnStart: 1,
     },
     'Keyboard Interactions': {
       rowStart: 2,
-      numberOfRows: 1,
+      numberOfRows: 2,
+      columnStart: 2,
     },
     'Additional Resources': {
-      rowStart: 2,
-      numberOfRows: 1,
+      rowStart: 4,
+      numberOfRows: 3,
     },
   },
   dialog: {
-    'Keyboard Interactions': {
+    'Focus Expectations': {
       rowStart: 1,
       numberOfRows: 2,
-    },
-    'Focus Expectations': {
-      rowStart: 3,
-      numberOfRows: 1,
-      columnStart: 2,
+      columnStart: 1,
     },
     'Semantic Elements': {
       rowStart: 1,
       numberOfRows: 1,
+      columnStart: 2,
     },
-    'Aria Attributes': {
+    'ARIA Roles and Attributes': {
       rowStart: 3,
-      numberOfRows: 1,
+      numberOfRows: 4,
       columnStart: 1,
     },
-    'Additional Resources': {
+    'Keyboard Interactions': {
       rowStart: 2,
+      numberOfRows: 3,
+      columnStart: 2,
+    },
+    'Additional Resources': {
+      rowStart: 3,
```

Comment:
Each specification block, for each component, has been hard-coded with where each block is supposed to start and how many rows/columns it spans. 

Just to make sure I'm not misunderstanding- this does not use the dynamic rowStart function you showed me, correct? I hope I didn't miss it anywhere!

Reply:
You are correct. We are doing this non-dynamically b/c we want the content blocks to always occur in the same order so it's all got to be hard-coded in.